contracts/base/LendingPool.sol

```diff
@@ -72,6 +72,13 @@ contract LendingPool is Base, LendingPoolInterface {
         whenNotPaused()
         whenLendingPoolNotPaused(address(this))
     {
+        uint256 maxTotalValueLocked = _getSettings().getPlatformSettingValue(
+            _getSettings().consts().MAXIMUM_TOTAL_VALUE_LOCKED()
+        );
+        require(
+            lendingToken.balanceOf(address(this)).add(amount) <= maxTotalValueLocked,
+            "TVL_MAXED"
+        );
```

Comment:
this only works when the lending token is not deposited to Compound. Instead, for now, we should be using the MarketState value. 

Also, because we are just launching with one market, this is fine, but if we were to have more markets, we would want to convert the value of the asset into 1 of the "USD value" tokens. I.e. convert DAI into USDC and use USDC as the normalized value for the max cap.

Reply:
Also, I think the setting is not considering the decimals.